client/src/pages/Profile.tsx

```
@@ -197,265 +227,345 @@ const Dashboard = () => {
+								</Card>
+								{/* GitHub Stats */}
+								<Card>
+									<CardHeader>
+										<CardTitle className="text-lg">GitHub Stats</CardTitle>
+									</CardHeader>
+									<CardContent className="space-y-8">
+										<div>
+											<h3 className="font-semibold mb-2">GitHub Streak Stats</h3>
+											{streakStat ? (
+													<div
+														dangerouslySetInnerHTML={{ __html: streakStat }}
```

Biome

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)